server/views/pages/index.njk

```diff
@@ -3,72 +3,61 @@
 {% from "govuk/components/table/macro.njk" import govukTable %}
 {% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
 
+{%- set dayjs = dayjs -%}
+{% set cases = unallocatedCases | length  %}
 {% set pageTitle = applicationName + " - Home" %}
+{% set tableData = {
+    caption: 'Unallocated community cases',
+    captionClasses: 'govuk-heading-l',
+    head: [
+        { text: 'Name / CRN' },
+        { text: 'Tier' },
+        { text: 'Sentence date' },
+        { text: 'Initial appointment' },
+        { text: 'Status' },
+        { text: 'Action' }
+    ],
+    rows: []
+} %}
+
+{% for item in unallocatedCases %}
+    {%- set tableRow = [
+        { text: item.name },
+        { text: item.tier },
+        { text: dayjs(item.sentenceDate).format('D MMM YYYY') },
+        { text: dayjs(item.initialAppointment).format('D MMM YYYY') },
+        { text: item.status },
+        { text: ('<a href="/" class="govuk-!-font-weight-bold govuk-link govuk-link--no-visited-state" aria-label="' + a11yTitle + '">' + "View case" + '</a>' ) | safe}
+    ] -%}
+    {{ tableData.rows.push(tableRow) }}
+
+{% endfor %}
 
 {% block content %}
 
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-full">
         <span class="govuk-caption-xl">Gateshead and South Tyneside</span>
         <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">South Tyneside</h1>
-      </div>
-    </div>
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-full">
+
         {{ mojSubNavigation({
           label: 'Sub navigation',
           items: [{
-            text: 'Unallocated cases (' + unallocatedCases + ')' if unallocatedCases else 'Unallocated cases',
+            text: 'Unallocated cases' if not cases else ('Unallocated cases' + (' (' + cases + ')' if cases < 99 else ' (99+)')),
```

Comment:
Here it is again - ideally this rule about 99+ would only exist in one place

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Reply:
Yep agree, I'll update 👍